src/controllers/softtakeover.cpp

```diff
@@ -104,29 +103,29 @@ bool SoftTakeover::ignore(ControlObject* control, double newParameter) {
      *          (regardless of the new value's arrival time)
      *      - the new value arrives awhile after the previous one
      *          (regardless of what the previous value was)
-     * 
+     *
      * Sheesh, this is much easier to show in a truth table!
-     * 
+     *
      * Sides    prev distance   new distance    new value arrives   Ignore
      * opposite close           far             later               TRUE
      * opposite far             far             later               TRUE
      * same     close           far             later               TRUE
      * same     far             far             soon                TRUE
      * same     far             far             later               TRUE
-     * 
+     *
      *      Don't ignore in every other case.
      */
 
-    qint64 currentTime = Time::elapsedMsecs();
+    mixxx::Duration currentTime = Time::elapsed();
     // We will get a sudden jump if we don't ignore the first value.
-    if (m_time == 0) {
+    if (m_time == mixxx::Duration::fromMillis(0)) {
```

Comment:
is not an important issue for me so it is OK to keep this.
It only jumps to my eyes because there is no need to have a unit for 0 and  mixxx::Duration(0) is shorter. 
fromMillis() will result into a multiplication by 0, but since the compiler will inline the code, it will be optimized away.